### ISSUE TITLE
LIG with ConvertTo-HPOVPowerShellScript -export misses LIG creation

### ISSUE THREAD

**Issue by maheinri** [HewlettPackard/POSH-HPOneView#428]
Hi when exporting a LIG to file it misses the LIG creation part:
Get-HPOVLogicalInterconnectgroup  |ConvertTo-HPOVPowerShellScript
outputs the command for the LIG creation, plus one for each Uplinkset  as expected.
If you use it with -export it, you only get the commands for the uplinkset  (see below)

BTW if you have multible networks, than the uplinktype is exported as  "tagged" but should be "ethernet" (see Network_all below)

### Expected Behavior
no difference between stdout and exporting it to file.

### Steps to reproduce
```powershell
PS C:\Users\maheinri\Desktop> Get-HPOVLogicalInterconnectgroup  |ConvertTo-HPOVPowerShellScript
# -------------- Attributes for Logical Interconnect Group "LIG_VC_F8_A"
$name                       = "LIG_VC_F8_A"
$frameCount                 = 1
$bayConfig                  = @{
        Frame1 = @{
                Bay3 = 'SEVC40f8'
        }
}
$redundancyType             = "NonRedundantASide"
$fabricModuleType           = "SEVC40f8"
$InterconnectBaySet         = 3
$networkLoopProtection      = $True
$LDPtagging                 = $True
#-- Generating snmp object for LIG
$readCommunity              = ""
$snmpConfig                += New-HPOVSnmpConfiguration -ReadCommunity $readCommunity -SnmpV3 -TrapDestinations $trapDes
tinations
New-HPOVLogicalInterconnectGroup -Name $name -Bays $bayConfig -FabricModuleType $fabricModuleType -FrameCount $frameCoun
t -InterConnectBaySet $InterconnectBaySet -enablenetworkLoopProtection:$networkLoopProtection -EnableLLDPTagging:$LDPtag
ging

# -------------- Attributes for Uplink Set "Fabric-A" associated to LogicalInterconnectGroup "LIG_VC_F8_A"
$name                       = "Fabric-A"
$uplinkType                 = "FibreChannel"
$parentName                 = "LIG_VC_F8_A"
$parent                     = Get-HPOVLogicalInterconnectGroup -Name $parentName
$uplinkPorts                = "Enclosure1:Bay3:Q5.1"
$fcSpeed                    = "Auto"
New-HPOVUplinkSet -InputObject $parent -Name $name -Type $uplinkType -fcUplinkSpeed $FCSpeed -UplinkPorts $uplinkPorts
# -------------- Attributes for Uplink Set "Fabric-B" associated to LogicalInterconnectGroup "LIG_VC_F8_A"
$name                       = "Fabric-B"
$uplinkType                 = "FibreChannel"
$parentName                 = "LIG_VC_F8_A"
$parent                     = Get-HPOVLogicalInterconnectGroup -Name $parentName
$uplinkPorts                = "Enclosure1:Bay3:Q6.1"
$fcSpeed                    = "Auto"
New-HPOVUplinkSet -InputObject $parent -Name $name -Type $uplinkType -fcUplinkSpeed $FCSpeed -UplinkPorts $uplinkPorts
# -------------- Attributes for Uplink Set "Network_All" associated to LogicalInterconnectGroup "LIG_VC_F8_A"
$name                       = "Network_All"
$uplinkType                 = "Tagged"
$parentName                 = "LIG_VC_F8_A"
$parent                     = Get-HPOVLogicalInterconnectGroup -Name $parentName
$uplinkPorts                = "Enclosure1:Bay3:Q1.1", "Enclosure1:Bay3:Q2.1"
$networks                   = "iSCSI", "MCTS-LAN", "Backup", "Fault-Tolerance", "HANA-Replication", "Cluster-Heartbeat",
 "VMotion", "HP-Intranet" | Get-HPOVNetwork
$ethMode                    = "Auto"
$lacpTimer                  = "Long"
New-HPOVUplinkSet -InputObject $parent -Name $name -Type $uplinkType -Networks $networks -EthMode $ethMode -lacptimer $l
acpTimer -UplinkPorts $uplinkPorts


PS C:\Users\maheinri\Desktop> Get-HPOVLogicalInterconnectgroup  |ConvertTo-HPOVPowerShellScript -export  C:\Users\maheinri\Desktop\test2.ps1

PS C:\Users\maheinri\Desktop> type .\test2.ps1
# -------------- Attributes for Uplink Set "Fabric-A" associated to LogicalInterconnectGroup "LIG_VC_F8_A"
$name                       = "Fabric-A"
$uplinkType                 = "FibreChannel"
$parentName                 = "LIG_VC_F8_A"
$parent                     = Get-HPOVLogicalInterconnectGroup -Name $parentName
$uplinkPorts                = "Enclosure1:Bay3:Q5.1"
$fcSpeed                    = "Auto"
New-HPOVUplinkSet -InputObject $parent -Name $name -Type $uplinkType -fcUplinkSpeed $FCSpeed -UplinkPorts $uplinkPorts
# -------------- Attributes for Uplink Set "Fabric-B" associated to LogicalInterconnectGroup "LIG_VC_F8_A"
$name                       = "Fabric-B"
$uplinkType                 = "FibreChannel"
$parentName                 = "LIG_VC_F8_A"
$parent                     = Get-HPOVLogicalInterconnectGroup -Name $parentName
$uplinkPorts                = "Enclosure1:Bay3:Q6.1"
$fcSpeed                    = "Auto"
New-HPOVUplinkSet -InputObject $parent -Name $name -Type $uplinkType -fcUplinkSpeed $FCSpeed -UplinkPorts $uplinkPorts
# -------------- Attributes for Uplink Set "Network_All" associated to LogicalInterconnectGroup "LIG_VC_F8_A"
$name                       = "Network_All"
$uplinkType                 = "Tagged"
$parentName                 = "LIG_VC_F8_A"
$parent                     = Get-HPOVLogicalInterconnectGroup -Name $parentName
$uplinkPorts                = "Enclosure1:Bay3:Q1.1", "Enclosure1:Bay3:Q2.1"
$networks                   = "iSCSI", "MCTS-LAN", "Backup", "Fault-Tolerance", "HANA-Replication", "Cluster-Heartbeat",
 "VMotion", "HP-Intranet" | Get-HPOVNetwork
$ethMode                    = "Auto"
$lacpTimer                  = "Long"
New-HPOVUplinkSet -InputObject $parent -Name $name -Type $uplinkType -Networks $networks -EthMode $ethMode -lacptimer $l
acpTimer -UplinkPorts $uplinkPorts
PS C:\Users\maheinri\Desktop>
```
### Version Information
```powershell
synergy01.mcts.local                LibraryVersion Path
--------------------                -------------- ----
ApplianceVersion: 5.00.00.404934.02 5.0.2226.2609  C:\Program Files\WindowsPowerShell\Modules\HPOneView.500\5.0.2226.
Name                           Value
----                           -----
PSVersion                      5.1.14409.1018
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.14409.1018
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

**Comment by ChrisLynchHPE** [comment]
Thanks for pointing this out.  Will get this addressed in a future update.

**Comment by ChrisLynchHPE** [comment]
This is fixed in [Release 5.0.2295.3359](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v5.0.2295.3359) and in [PowerShell Gallery](https://www.powershellgallery.com/packages/HPOneView.500/5.0.2295.3359).

Please verify before this issue is closed.

**Comment by ChrisLynchHPE** [comment]
Closing due to no further activity.  If you wish, feel free to re-open to discuss further.
